src/tests/testAuthConfig.cc

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 1996-2021 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+
+#if USE_AUTH
+
+#include "auth/Config.h"
+#include "tests/testAuthConfig.h"
+#include "unitTestMain.h"
+
+CPPUNIT_TEST_SUITE_REGISTRATION( testAuthConfig );
+
+void
+testAuthConfig::create()
+{
+    {
+        Auth::Config local;
+    }
+
+    {
+        auto *dynamic = new Auth::Config;
+        CPPUNIT_ASSERT(dynamic != nullptr);
```

Comment:
Please remove. We should not be testing C++ guarantees because testing them creates various unnecessary questions for the reader: Is this some kind of poorly converted C code that I should fix? Was this code written by a C developer and, hence, is likely to contain other problems I should be on the lookout for? Am I misreading something an `dynamic` can actually be nil here? Etc., etc.
```suggestion
```